### PR TITLE
Made enchanted books source more consistent and some small fixes

### PR DESCRIPTION
**Pull request by MrNightmare03**
Feel free to comment if anything looks wrong

### CODE REVIEW

File: items/FIRE_ASPECT;3.json [hunk 1]
```diff
@@ -12,11 +12,19 @@
     "§7Use this on an item in an Anvil",
     "§7to apply it!",
     "",
+    "§6Source:",
+    "§aIII: §7Slayer",
```
Comment (jani270) [inline items/FIRE_ASPECT;3.json:16]: Might wanna say which slayer (like it does on duplex)

**Comment by jani270** [comment]
While you are at it if you want can you add the stuff to: Hecatomb, delicate and champion

**Comment by MrNightmare03** [comment]
> While you are at it if you want can you add the stuff to: Hecatomb, delicate and champion

Anything more besides source and what to apply it to?

**Comment by jani270** [comment]
> > While you are at it if you want can you add the stuff to: Hecatomb, delicate and champion
> 
> Anything more besides source and what to apply it to?

Dont think so

**Comment by MrNightmare03** [comment]
Source*

**Review by NopoTheGamer — APPROVED** [review]

**Review by jani270 — APPROVED** [review]

**Pull request merged**